book/src/volume-health-monitor.md

```diff
@@ -0,0 +1,34 @@
+# Volume Health Monitory Feature
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | CSI external-health-monitor-controller sidecar Version | CSI external-health-monitor-agent sidecar Version
+--|--|--|--|--
+Alpha | 1.19 | | 0.1.0 | 0.1.0
+
+## Overview
+
+The External Health Monitor is part of Kubernetes implementation of [Container Storage Interface (CSI)](https://github.com/container-storage-interface/spec). It was introduced as an Alpha feature in Kubernetes v1.19.
+
+The [External Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) is impelmeted as two components: `External Health Monitor Controller` and `External Health Monitor Agent`.
+
+- External Health Monitor Controller:
+  - The external health monitor controller will be deployed as a sidecar together with the CSI controller driver, similar to how the external-provisioner sidecar is deployed.
+  - Trigger controller RPC to check the health condition of the CSI volumes.
+  - The external controller sidecar will also watch for node failure events. This component can be enabled via a flag.
```

Comment:
Can you explain what happens in the case of a node failure? Also exactly which flag?

Reply:
Done